src/mito2/src/config.rs

```diff
@@ -193,13 +193,13 @@ impl MitoConfig {
 
     fn adjust_buffer_and_cache_size(&mut self, sys_memory: ReadableSize) {
         // Use 1/64 of OS memory as global write buffer size, it shouldn't be greater than 1G in default mode.
```

Comment:
Also change the doc here: 
```suggestion
        // Use 1/8 of OS memory as global write buffer size, it shouldn't be greater than 1G in default mode.
```

Reply:
I think it's better to remove these comments and add some constants for these factors.

Reply:
Move the comments to the constants.